static/src/javascripts.flow.archive/bootstraps/enhanced/common.js

```diff
@@ -54,7 +54,6 @@ import { init as initRelativeDates } from 'common/modules/ui/relativedates';
 import { smartAppBanner } from 'common/modules/ui/smartAppBanner';
```

Comment:
Anything under `javascript.flow.archive` directory can remain untouched as it just serves as a snapshot for flow types

Reply:
Yes, but what would be the benefit for not removing code that is deleted in this case? 
I find it easier to remove all references to `pinterest`.